src/sets/liver_livr_rules.erl

```diff
@@ -30,6 +30,7 @@
 -export([email/3]).
 -export([url/3]).
 -export([iso_date/3]).
+-export([iso_datetime/3]).
```

Comment:
The error for this func will always be {error, wrong_datetime}

Reply:
will restrict to wrong_time or format_error.